notebooks/sanity_checks_utils.py

```diff
@@ -62,46 +63,46 @@
             "ABCs",
             "GC_B (I)",
             "GC_B (II)",
-            "MNP/B doublets",
             "Memory B cells",
             "Naive B cells",
             "Pre-B",
-            "Pro-B",
+            "Pro-B"
         ],
         "Plasma": ["Plasma cells", "Plasmablasts"],
         "Mono": ["Classical monocytes", "Nonclassical monocytes"],
-        "Macro": [
-            "Alveolar macrophages",
-            "Erythrophagocytic macrophages",
-            "Intermediate macrophages",
-            "Intestinal macrophages",
-        ],
-        "T": [
-            "MAIT",
-            "MNP/T doublets",
-            "T_CD4/CD8",
-            "Teffector/EM_CD4",
-            "Tem/emra_CD8",
-            "Tfh",
-            "Tgd_CRTAM+",
-            "Tnaive/CM_CD4",
-            "Tnaive/CM_CD4_activated",
-            "Tnaive/CM_CD8",
-            "Tregs",
-            "Trm/em_CD8",
-            "Trm_Tgd",
-            "Trm_Th1/Th17",
-            "Trm_gut_CD8",
-        ],
+        "CD8 T": ["Tem/emra_CD8", "Tnaive/CM_CD8", "Trm/em_CD8", "Trm_gut_CD8"],
+        "CD4 T":["Teffector/EM_CD4", "Tfh", "Tnaive/CM_CD4", "Tnaive/CM_CD4_activated", "Tregs", 
+                 "Trm_Th1/Th17"],
+        "T": ["MAIT","T_CD4/CD8","Tgd_CRTAM+","Trm_Tgd"],
         "NK": ["NK_CD16+", "NK_CD56bright_CD16-"],
-        "ICL3": ["ILC3"],
         "DC": ["DC1", "DC2", "migDC", "pDC"],
         "Mast": ["Mast cells"],
         "Red_blood": ["Erythroid"],
         "Bone_marrow": ["Megakaryocytes"],
         "Non-differentiated": ["Progenitor"],
-        "To remove": ["Cycling", "T/B doublets", "Cycling T&NK"],
+        "To remove":  ["Cycling", "T/B doublets", "Cycling T&NK",
```

Comment:
Is this going to be updated after your conversation with Sabrina ?

Reply:
Yes, just CD8 T -> CD8T and CD4 T -> CD4T